src/config/config.service.ts

```diff
@@ -77,8 +77,8 @@ export class ConfigService {
     return this.providerMap.get(id.toString())?.userGraphEndpoint!;
   }
 
-  public providerApiToken(id: ProviderId | AnyNumber): string {
-    return this.providerMap.get(id.toString())?.apiToken!;
+  public providerAccessToken(): string {
+    return this.nestConfigService.get<string>('PROVIDER_ACCESS_TOKEN')!;
```

Comment:
This was using `providerMap` because I was planning for the eventuality that at some point we would support multiple providers with a single instance of this app.

Reply:
AH oops, missed that. I'll shore this back up